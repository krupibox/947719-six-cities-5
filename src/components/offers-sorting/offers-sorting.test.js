import OffersSorting from './offers-sorting';

describe(`<OffersSorting/>`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
            .create(
                <OffersSorting
                  sortingType={`Popular`}
                  onTypeClick={() => {}}
                />)
            .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
