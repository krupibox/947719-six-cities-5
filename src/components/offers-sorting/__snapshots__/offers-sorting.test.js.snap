// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OffersSorting/> should render correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <span
    className="places__sorting-type"
    onClick={[Function]}
    tabIndex={0}
  >
    Popular
    <svg
      className="places__sorting-arrow"
      height={4}
      width={7}
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
</form>
`;
